.travis.yml

```diff
@@ -19,6 +19,9 @@ script:
 - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
     cargo bench --verbose --no-run;
   fi
+- if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
```

Comment:
Rudy is nightly-only: [src/lib.rs +4-6](https://github.com/adevore/rudy/blob/d3d97233e51cdc9714ecfeffa589fb7028ee7cec/src/lib.rs#L4-L6)